source/Nevermore/IReadQueryExecutor.cs

```diff
@@ -804,5 +804,8 @@ public interface IReadQueryExecutor
         /// <param name="cancellationToken">Token to use to cancel the command.</param>
         /// <returns>A data reader.</returns>
         [Pure] Task<DbDataReader> ExecuteReaderAsync(PreparedCommand preparedCommand, CancellationToken cancellationToken = default);
+
+        // TODO: work out a way to avoid needing to expose this
```

Comment:
I think this comment is no longer relevant; we decided to go with exposing this since it's called from our querybuilders.